ui/src/pages/Tenant/Detail/NewBackup/index.tsx

```diff
@@ -29,8 +29,11 @@ export default function NewBackup() {
   const scheduleValue = Form.useWatch(['scheduleDates'], form);
 
   const distType = [
```

Comment:
This seems to be a typo. Maybe `destTypes` is better.

Reply:
done